### PR TITLE
Add side menu static screens

### PR DESCRIPTION
**Pull request by zohaibahmed-22**

### CODE REVIEW

File: src/config/check_iphone_x.ts [hunk 1]
```diff
@@ -3,4 +3,4 @@ import { Platform, Dimensions } from 'react-native';
 const deviceHeight = Dimensions.get('window').height;
 const deviceWidth = Dimensions.get('window').width;
 export const isIphoneX =
-  Platform.OS === 'ios' && deviceHeight <= 812 && deviceWidth <= 375;
+  Platform.OS === 'ios' && deviceHeight >= 812 && deviceWidth >= 375;
```
Comment (anandwahed) [inline src/config/check_iphone_x.ts]: Does the above constraints work for the new `iPhone XS Max` also ?

File: src/screens/sidebar/about/investment_agreement_screen.tsx [hunk 2]
```diff
@@ -1,21 +1,68 @@
...
-          Investment Agreement Screen coming soon
-        </Text>
-      </View>
+      <ScrollView style={styles.investmentAgreementView}>
+        <View style={{ marginLeft: 25, marginRight: 25 }}>
+          <Text style={styles.investmentAgreementText}>
+            Investment Agreement
+          </Text>
+          <Text style={styles.sedUtPerspiciatisText}>
+            Sed ut perspiciatis unde omnis iste natus error sit voluptatem
```
Comment (anandwahed) [inline src/screens/sidebar/about/investment_agreement_screen.tsx]: This data might probably come from any API end-point i guess. Better to move these text to a JSON file in the `data` folder and fetch it here.

File: src/config/check_iphone_x.ts (hunk 1 above)
Comment (zohaibahmed-22) [reply src/config/check_iphone_x.ts]: Yes, tried the simulator

File: src/screens/sidebar/about/investment_agreement_screen.tsx (hunk 2 above)
Comment (zohaibahmed-22) [reply src/screens/sidebar/about/investment_agreement_screen.tsx]: okay will update

**Pull request merged**